### DISCUSSION TITLE
3.0.0 doesn’t work in Orion

### DISCUSSION THREAD

**Discussion by amarjt** [coddingtonbear/obsidian-web#142] [Exotic Browser Support]
Steps to reproduce:

Install [Obsidian Web Extension](https://chrome.google.com/webstore/detail/obsidian-web/edoacekkjanmingkbkgjndndibhkegad) from the Google store
Install the [Local REST API extension](https://github.com/coddingtonbear/obsidian-local-rest-api) in Obsidian
Copy the API key from the Obsidian Local REST API extension into Obsidian Web extension.
Turn HTTPS in Obsidian Local REST API off, because Orion doesn't trust the certificate.
Try to clip a page or selection. Note doesn't show up in the Obsidian Vault.
Expected behavior:
That a web clipping would show up in my vault.
I tried and tested this in Arc and Chrome following the steps above and it works.
One of the problems could be that it looks like the Save button in the Obsidian Web extension doesn't save anything.

Extension type:
Version: 3.0.1
Updated: November 15, 2023

**Comment by daniel347x** [comment]
I'm having the same problem, only in Chrome (and with both HTTPS and HTTP enabled). Was working a couple days ago.

**Comment by amarjt** [comment]
> I'm having the same problem, only in Chrome (and with both HTTPS and HTTP enabled). Was working a couple days ago.

Is it not working ONLY in chromium based browser like Google Chrome and MS Edge?

**Comment by coddingtonbear** [comment]
I’m afraid I can’t offer support for anything other than Chrome; so I’ve moved this into a discussion so you might be able to find somebody in the community to help.
